android/app/src/main/java/network/mysterium/registration/RegistrationTopupSelectCurrencyFragment.kt

```diff
@@ -0,0 +1,99 @@
+package network.mysterium.registration
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import network.mysterium.payment.Currency
+import network.mysterium.vpn.databinding.RegistrationPaymentCryptoListItemBinding
+
+class RegistrationTopupSelectCurrencyFragment : Fragment() {
+
+    private lateinit var viewModel: RegistrationViewModel
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+//        val binding = FragmentRegistrationTopupSelectCurrencyBinding.inflate(inflater, container, false)
```

Comment:
Why so big commented code chunk?

Reply:
As I mentioned in the description, registration flow is left intact, but unused.